test/integration/features/cmd-openshift.feature

```diff
@@ -25,6 +25,12 @@ cluster in VM provided by Minishift.
        And stdout should contain "kubernetes"
        And stdout should contain "router"
 
+  Scenario: Pods docker-registry and router are ready before OpenShift restart
+     Given user waits "30" seconds
```

Comment:
@agajdosi Why do we need to wait here?

Reply:
@LalatenduMohanty This is because nodes need some time to get not only running, but be also ready. The second sleep is because right after restart, oc sometimes gives TLS error (the after restart sleep could be probably shorter), but to be sure 30 seconds works best.

Reply:
if #2169 is fixed then this can be removed i guess